src/environments/environment.ts

```diff
@@ -6,7 +6,6 @@ export const environment = {
   production: false,
   recaptchaSiteKey: '6LdX85EUAAAAAGeujVRsy8KJzRvYoC-InusmiDZr',
   apiEndpoint: 'https://api.prylabs.net',
```

Comment:
Same here

Reply:
I believe the api.prylabs.net is for the faucet though

Reply:
oh yeah